### DISCUSSION TITLE
Documentation issues/PRs repo

### DISCUSSION THREAD

**Discussion by theca11** [elgatosf/elgato#43] [Ideas]
[Docs](https://docs.elgato.com/sdk/plugins/overview) are in the most part extensive and accurate, but there are some bugs/innacuracies and things that could be improved. Right now there is no really suitable place to report issues or make PRs. It would be nice to have a dedicated repo for this, even your own docs repo if you're willing to share it publicly.

**Comment by GeekyEggo** [comment]
Hey @theca11, we agree, and we want our docs to be excellent. To achieve this, later this year we plan to completely overhaul our existing documentation content and structure. In addition, we're hoping to open-source the repository that drives the documentation and enable pull-requests for those who wish to contribute. Until then, our GitHub discussions remain the best place to provide any feedback and suggestions.

We appreciate your interest in contributing to our docs. 🙌
